docs/learning/metadata/index.md

```diff
@@ -0,0 +1,21 @@
+# Metadata packages in the iTwin.js Library
+
+## Packages
+
+The metadata packages implement the abstract concepts of EC in typescript [EC overview](../../bis/ec/index.md)
+The following packages exist:
+
+- ($ecschema-metadata) is the basic package that exposes metadata objects.
+- ($ecschema-editing) Since the basic package only implements an API for understanding schemas, this package adds editing on top of it.
+- ($ecschema-locaters) contains classes for locating and loading EC schema files from the file system.
+- ($ecschema-rpc) provides a schema RPC interface implementation
```

Comment:
should this line be split into 2? the 2 npm packages users would want are `ecschema-rpcinterface-impl` and `ecschema-rpcinterface-common`

Reply:
Ah, I never used these so I didn't know. I still need to make sure those variables resolve correctly, the documentation process could use some documentation. I think it's in `docs/config/docSites.json` but I'm not sure yet

Reply:
I ended up removing those packages as I noticed we do not publish API docs for them. It may be more confusing than good to have them listed here.